lavague-core/lavague/core/navigation.py

```diff
@@ -428,15 +451,16 @@ def execute_instruction(self, instruction: str) -> ActionResult:
         success = False
         action_full = ""
 
-        rephrased_query = self.rephraser.rephrase_query(instruction)
-
+        action = self.rephraser.rephrase_query(instruction)
```

Comment:
This part could remain untouched - rephraser will be removed soon so it will be easier to merge if there's minimal impact on it